Source/PlasticSourceControl/Private/PlasticSourceControlOperations.cpp

```diff
@@ -23,6 +23,7 @@
 
 #include "ISourceControlModule.h"
 #include "SourceControlOperations.h"
+#include <PlasticSourceControlShell.h>
```

Comment:
It would be a bit better to follow the conventions used in Unreal:
Replace the brackets with double quotes and put it above alongside all the others PlasticSourceControlXxx.h includes :)
#include "PlasticSourceControlShell.h"

Reply:
Fixed in https://github.com/PlasticSCM/UEPlasticPlugin/pull/96/commits/2c35a0ceb84a6fd47b04b59bef40ec30f2e8e4fe

Thanks for the reminder!